lib/jsonld_serializers.js

```diff
@@ -204,13 +223,7 @@ class ResourceSerializer extends JsonLdItemSerializer {
   statements () {
     var stmts = JsonLdItemSerializer.prototype.statements.call(this)
 
-    if (this.body.identifier) {
-      stmts = this._bubbleIdentifiersUpToStatements({ acqnum: 'idAcqnum', barcode: 'idBarcode',
-      bnum: 'idBnum', callnum: 'idCallNum', catnyp: 'idCatnyp', dcc: 'idDcc',
-      exhibition: 'idExhib', hathi: 'idHathi', isbn: 'idIsbn', issn: 'idIssn',
-      lcc: 'idLcc', lccc: 'idLccCoarse', mmsdb: 'idMmmsDb', mss: 'idMss', objnum: 'idObjNum',
-      oclc: 'idOclc', rlin: 'idRlin', uuid: 'idUuid' }, stmts)
-    }
```

Comment:
Previously, this block caused identifiers with the named namespaces (e.g. 'barcode') to be *copied* to properties (e.g. 'idBarcode'). This was a temporary measure to cause those 'id*' values to be populated in lieu of adding them to the actual ES index, which we're now doing. Note many of these (e.g. hathi) have not been mapped.